bogenliga/src/app/modules/user/services/login-data-provider.service.ts

```diff
@@ -89,4 +92,17 @@ export class LoginDataProviderService extends DataProviderService {
           }
         });
   }
+
+  public getDefaultUser() : CredentialsDO {
```

Comment:
Methode nicht public! Den Default User geht niemanden außer den login-data-provider was an (Geheimnisprinzip!).
v.a. hast du sie ja auch als Konstante in der signInDefaultUser Methode (so wie es sein soll)